packages/frontend-2/pages/server-management/active-users.vue

```diff
@@ -0,0 +1,396 @@
+          onClick: closeChangeUserRoleDialog
+        }
+      ]"
+    />
+  </div>
+</template>
+
+<script setup lang="ts">
+import { ref } from 'vue'
+import { useQuery, useMutation } from '@vue/apollo-composable'
+import { debounce, isArray } from 'lodash-es'
+import { useGlobalToast, ToastNotificationType } from '~~/lib/common/composables/toast'
+import { useLogger } from '~~/composables/logging'
+import { InfiniteLoaderState } from '~~/lib/global/helpers/components'
+import { Nullable, ServerRoles, Optional } from '@speckle/shared'
+import { graphql } from '~~/lib/common/generated/gql'
+import { ItemType, UserItem } from '~~/lib/server-management/helpers/types'
+import { isUser } from '~~/lib/server-management/helpers/utils'
+import { useActiveUser } from '~~/lib/auth/composables/activeUser'
+import {
+  MagnifyingGlassIcon,
+  ShieldExclamationIcon,
+  ShieldCheckIcon,
+  TrashIcon
+} from '@heroicons/vue/20/solid'
+import {
+  convertThrowIntoFetchResult,
+  getCacheId,
+  getFirstErrorMessage,
+  updateCacheByFilter
+} from '~~/lib/common/helpers/graphql'
+
+definePageMeta({
+  middleware: ['admin']
+})
+
+const getUsers = graphql(`
+  query AdminPanelUsersList($limit: Int!, $cursor: String, $query: String) {
+    admin {
+      userList(limit: $limit, cursor: $cursor, query: $query) {
+        totalCount
+        cursor
+        items {
+          id
+          email
+          avatar
+          name
+          role
+          verified
+          company
+        }
+      }
+    }
+  }
+`)
+
+const adminDeleteUser = graphql(`
+  mutation AdminPanelDeleteUser($userConfirmation: UserDeleteInput!) {
+    adminDeleteUser(userConfirmation: $userConfirmation)
+  }
+`)
+
+const changeRoleMutation = graphql(`
+  mutation AdminChangeUseRole($userRoleInput: UserRoleInput!) {
+    userRoleChange(userRoleInput: $userRoleInput)
+  }
+`)
+
+const { triggerNotification } = useGlobalToast()
+const logger = useLogger()
+const { activeUser } = useActiveUser()
+
+const userToModify: Ref<Nullable<UserItem>> = ref(null)
+const searchString = ref('')
+const showUserDeleteDialog = ref(false)
+const showChangeUserRoleDialog = ref(false)
+const newRole = ref<ServerRoles>()
+const oldRole = computed(() => userToModify.value?.role as Optional<ServerRoles>)
+const infiniteLoaderId = ref('')
+
+const queryVariables = computed(() => ({
+  limit: 50,
+  query: searchString.value
+}))
+
+const moreToLoad = computed(
+  () =>
+    !extraPagesResult.value?.admin?.userList ||
+    extraPagesResult.value.admin.userList.items.length <
+      extraPagesResult.value.admin.userList.totalCount
+)
+
+const users = computed(() => extraPagesResult.value?.admin.userList.items || [])
+
+const isCurrentUser = (userItem: UserItem) => {
+  return userItem.id === activeUser.value?.id
+}
+
+const {
+  result: extraPagesResult,
+  fetchMore: fetchMorePages,
+  variables: resultVariables,
+  onResult
+} = useQuery(getUsers, queryVariables)
```

Comment:
Search is busted, it appears that something is setting the actual KeyboardEvent object instead of the new search string into `searchString`

<img width="808" alt="image" src="https://github.com/specklesystems/speckle-server/assets/938316/3d8f16fe-7a0c-49a7-8c1b-00479bce7a88">
<img width="822" alt="image" src="https://github.com/specklesystems/speckle-server/assets/938316/b61c0890-28a1-4880-b2f8-8e9317b99186">